### ISSUE TITLE
Failed to activate the auto-copyright package

### ISSUE THREAD

**Issue by robhicks** [lee-dohm/auto-copyright#9]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.196.0
**System**: Mac OS X 10.10.3
**Thrown From**: [auto-copyright](https://github.com/lee-dohm/auto-copyright) package, v0.4.0

### Stack Trace

Failed to activate the auto-copyright package

```
At 0 is not a valid selector in /Users/robhicks/.atom/packages/auto-copyright/package.json

SyntaxError: 0 is not a valid selector in /Users/robhicks/.atom/packages/auto-copyright/package.json
  at exports.validateSelector (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/clear-cut/index.js:115:19)
  at CommandRegistry.module.exports.CommandRegistry.add (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:44:9)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:735:70
  at Package.module.exports.Package.subscribeToActivationCommands (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:764:11)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/package.js:201:30
  at Package.module.exports.Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:147:15)
  at Package.module.exports.Package.activate (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:195:14)
  at PackageManager.module.exports.PackageManager.activatePackage (/Applications/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:21)
  at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:270:29
  at exit (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/settings-view/lib/package-manager.js:41:16)
  at triggerExitCallback (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:213:47)
  at ChildProcess.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/buffered-process.js:235:18)
  at emitTwo (events.js:87:13)
  at ChildProcess.emit (events.js:169:7)
  at Process.ChildProcess._handle.onexit (child_process.js:1057:12)

  at /Users/robhicks/.atom/packages/auto-copyright/package.json:1:1
```

### Commands

```
     -0:19.5.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-monokai-seti.theme-seti-ui)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "keybinding-resolver",
      "tabs",
      "tree-view",
      "language-objective-c",
      "autoflow",
      "metrics",
      "linter-jshint",
      "atom-jshint"
    ],
    "themes": [
      "seti-ui",
      "monokai-seti"
    ],
    "audioBeep": false,
    "excludeVcsIgnoredPaths": false
  }
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
Stylus, v1.0.0
Sublime-Style-Column-Selection, v1.2.3
angularjs, v0.1.0
atom-color-highlight, v3.0.9
atom-typescript, v4.1.0
autocomplete-paths, v1.0.2
autocomplete-plus, v2.12.1
autocomplete-snippets, v1.3.0
change-case, v0.5.1
color-picker, v1.7.0
compare-files, v0.4.1
editorconfig, v1.0.0
file-type-icons, v0.5.4
file-types, v0.3.0
filesize, v0.2.0
highlight-line, v0.10.2
highlight-selected, v0.9.2
jsformat, v0.8.0
jshint, v1.3.5
language-ejs, v0.1.0
language-jade, v0.4.0
last-cursor-position, v0.9.0
linter, v0.12.1
local-history, v3.1.0
minimap, v4.8.0
monokai-seti, v0.7.0
project-manager, v1.15.5
seti-ui, v0.6.3
show-in-folder, v0.1.0
sublime-tabs, v0.5.6
tabs-to-spaces, v0.9.2
view-tail-large-files, v0.1.14

# Dev
No dev packages
```

**Comment by lee-dohm** [comment]
Thanks. I'm looking at it now.

**Comment by lee-dohm** [comment]
Published in v0.4.1
